source/MaterialXGenOgsXml/OgsXmlGenerator.cpp

```diff
@@ -78,27 +78,13 @@ namespace
         { "u_time", "Time" }
     };
 
-    namespace OgsInputFlags
+    namespace OgsParameterFlags
     {
         static const std::string
             VARYING_INPUT_PARAM = "varyingInputParam",
             IS_REQUIREMENT_ONLY = "isRequirementOnly";
     }
 
-
-    // Custom flags required by OGS XML
-    static const StringMap OGS_FLAGS_MAP = 
-    {
-        { "Pw", OgsInputFlags::VARYING_INPUT_PARAM },
-        { "Pm", OgsInputFlags::IS_REQUIREMENT_ONLY },
-        { "Nw", OgsInputFlags::VARYING_INPUT_PARAM },
-        { "Nm", OgsInputFlags::IS_REQUIREMENT_ONLY },
-        { "Tw", OgsInputFlags::VARYING_INPUT_PARAM },
-        { "Tm", OgsInputFlags::IS_REQUIREMENT_ONLY },
-        { "Bw", OgsInputFlags::IS_REQUIREMENT_ONLY },
-        { "Bm", OgsInputFlags::IS_REQUIREMENT_ONLY }
-    };
```

Comment:
These were basically exceptions, `varyingInputParam` was already the default for vertex inputs.